index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MemoFibraGame</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>MEMO FIBRA GAME</h1>
+    <h2>WHAT FIBRA ARE YOU TODAY?</h2>
+    <div class="row" id="numbers">
+        <p id="p1" style="padding-right: 40px;">CLICKS: 0</p>
+        <p id="timer">TIME: 0</p>
+    </div>
+    <hr>
+    <div class="row">
+        <div class="card picture-hidden" onclick="onCardClicked(event)"></div>
+        <div class="card picture-hidden" onclick="onCardClicked(event)"></div>
+        <div class="card picture-hidden" onclick="onCardClicked(event)"></div>
+        <div class="card picture-hidden" onclick="onCardClicked(event)"></div>
```

Comment:
los eventos se asignan del lado de JavaScript para mantener toda la lógica de presentación junta.

Reply:
code in html changed for js code:
const $cardClicked = document.querySelectorAll('.card');
$cardClicked.forEach(($click) => {
    $click.addEventListener('click', onCardClicked);
  });